examples/cors/backend/front-envoy.yaml

```diff
@@ -21,21 +21,23 @@ static_resources:
             - name: www
               domains:
               - "*"
-              cors:
```

Comment:
does this have to be configured with `typed_per_filter_config` now ?

Reply:
Not `have to`. But the inline `cors` field is deprecated and the `typed_per_filter_config` is prefered.

Reply:
just to clarify my understanding - is it correct that we could put this config in the `envoy.filters.http.cors` http filter?

im asking partly because looking at the docs here https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/cors/v3/cors.proto#extensions-filters-http-cors-v3-cors it doesnt give much away

Reply:
Yeah, I am sure it's right. I deprecated this `cors` field in #23280